Makefile

```diff
@@ -15,9 +15,6 @@ lint-fix:
 	npm run lint-fix
 	npm run prettier-fix
 
-coverage:
-	npm run coverage
-
```

Comment:
Erkhm...
What? Why?

Reply:
The same target appears with the same rule in the makefile. When you run `make coverage`, make even reports it:
```
Makefile:19: warning: overriding recipe for target 'coverage'
Makefile:5: warning: ignoring old recipe for target 'coverage'
```
So I figured it must be some kind of mistake.

Reply:
you're totally right!
I checked preview and there was only one... but i forgot I was checking your changed file. Good job!